tsconfig.paths.json

```
@@ -0,0 +1,14 @@
+// tsconfig.paths.json
+
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "paths": {
+      "@/*": ["*"],
+      "@Assets/*": ["./Assets/*"],
+      "@Pages/*": ["./Pages/*"],
+      "@Components/*": ["./Components/*"],
+      "@Styles/*": ["./Styles/*"]
+    }
+  }
+}
```

다음과 같이 수정하여도 동일하게 동작하는 것을 확인하실 수 있습니다 :)

```suggestion
{
  "compilerOptions": {
    "baseUrl": "src",
    "paths": {
      "@/*": ["*"],
      "@Assets/*": ["Assets/*"],
      "@Pages/*": ["Pages/*"],
      "@Components/*": ["Components/*"],
      "@Styles/*": ["Styles/*"]
    }
  }
}

```